### ISSUE TITLE
Loss convergence problem

### ISSUE THREAD

**Issue by Hansheng-LIANG** [pengzhiliang/MAE-pytorch#103]
Hello, what is the approximate range that your loss converges to during the pre-training phase? I noticed that your training starter script is set to train for 1600 epochs, but in my training process, the loss oscillates around 0.6 after 200 epochs. Looking forward to reply.🙏
